src/Microsoft.DocAsCode.Build.Engine/HostServiceCreator.cs

```diff
@@ -72,10 +72,12 @@ public virtual bool ShouldProcessorTraceInfo(IDocumentProcessor processor)
             {
                 Logger.LogDiagnostic($"Processor {processor.Name}, File {file.FullPath}: Loading...");
 
-                metadata = ApplyFileMetadata(file.FullPath, metadata, fileMetadata);
+                var fileMeta = NeedApplyMetadata()
```

Comment:
`fileMeta` -> `fileMetadata`?

Reply:
already used in method's parameter list...